openpype/hosts/nuke/plugins/load/actions.py

```diff
@@ -17,7 +17,7 @@ class SetFrameRangeLoader(load.LoaderPlugin):
                 "yeticache",
                 "pointcache"]
     representations = ["*"]
-    extension = {"*"}
+    extensions = {"*"}
```

Comment:
Technically this would be redundant because the parent class sets the same value. Not a big issue - just wanted to point that out.

Reply:
well that explains, why it was working anyway :D